fractions/javaee/jaxrs-jsonb/src/main/resources/modules/org/wildfly/swarm/jaxrs/jsonb/main/module.xml

```diff
@@ -0,0 +1,5 @@
+<module xmlns="urn:jboss:module:1.3" name="org.wildfly.swarm.jaxrs.jsonb">
```

Comment:
I just wonder, would it make sense to document in the Thorntail docs when one fraction needs module.xml or module.conf or both ? I hinestly don't understand why the `jsonb` fraction needs a module.xml now, the tests are passing. I guess it is mainly fr consistency purposes but it is only a guess :-)

Reply:
I remember I forced you to add `module.conf` to the `jaxrs-jsonb` fraction, and now I'm moving it back to `module.xml`, oh the irony :-) So, indeed it's for consistency, but if `module.xml` is generated from `module.conf` by the fraction plugin, it contains more module dependencies. Those are not necessary in this case, as I tried to explain in the accompanying `no.module.conf`.

Reply:
I forgot about it, np :-). OK, so far as I understand now we'll have a more optimal module.xml for the `jaxrs-jsonb`

Reply:
I guess what I wanted to say is that we're always learning :-) I inspected all the fractions that don't have a `module.conf`, changed my mind about the `jaxrs-jsonb`, and added a `no.module.conf` file to all directories under `fractions/` that don't have a `module.conf`.

Reply:
Sounds good

Reply:
Thanks!